oeps/oep-0014-proc-archive-repos.rst

```diff
@@ -172,35 +180,24 @@ and notifying the open source community are essentially the same; the major
 differences from the proposed process are outlined below.
 
 
-Alternative 1: Transfer Repository
-----------------------------------
+Alternative 1: Archive In Place
+-------------------------------
 
-Transfer the obsolete repository to a new organization: edx-archived.
-
-.. note::
-    We now use the edx-unsupported organization for forks that we no longer
-    maintain.
+Use Github's archive feature and updated documentation to archive the repository
+in place.
 
 Pros:
 
-- `edx organization`_ is no longer littered with unsupported/obsolete
-  repositories.
-- GitHub search results within the `edx organization`_ do not include matches
-  in archived repositories. This could decrease confusion, especially since
-  repo descriptions are not included in search results.
-- Gemnasium monitoring may cease automatically (although this would need to be
-  confirmed).
-- Pattern followed by `Facebook`_, and thus might be familiar to others.
+- Old code hasn't moved so it can be easily found.
+- Old code can show up in searches to find historical context.
 
 Reasons rejected:
 
-- This creates another organization that edX must maintain and adds
-  administrative overhead.
-- It could be difficult for people to find the code through search, though
-  forwarding links would work for anyone who already linked to the repositories.
-
-.. _Facebook: https://github.com/facebookarchive
-
+- Through some experience with this method, we've learned that it's less
+  valuable than we expected.
+- Being able to know whether code is alive or dead is really helpful when making
+  major changes and if dead code can't easily be filtered from searches it
+  allows us to move faster.
```

Comment:
allows us to move faster -> slows us down
(since we're saying "**can't** easily be filtered")